app/components/CreatePasswordForm/components/Layout.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+
+import Page from 'components/Page'
+import styles from './index.css'
+
+class Layout extends Component {
+  render() {
+    const {
+      newPassword,
+      error,
+      updateNewPassword,
+    } = this.props
+
+    return (
+      <React.Fragment>
```

Comment:
After reading the name of this component I was expecting to see a form tag somewhere, or a Form from a library (redux-form, final-form...), but you are handling the data of the inputs yourself using ```updateNewPassword``` hook.

Which is defined 3 components up in the tree: ```route ChangePassword > layout ChangePassword >  container CreatePasswordForm > layout CreatePAsswordForm```

Despite the code is quite clear and nice, I see too many steps in the flow. Why not pass all the responsibility to CreatePassword component to handle everything by itself giving it powers to update the redux state?